Formula/astrometry-net.rb

```diff
@@ -0,0 +1,66 @@
+class AstrometryNet < Formula
+  include Language::Python::Virtualenv
+
+  desc "Automatic identification of astronomical images"
+  homepage "https://github.com/dstndstn/astrometry.net"
+  url "https://github.com/dstndstn/astrometry.net/releases/download/0.76/astrometry.net-0.76.tar.gz"
+  sha256 "244355ef9716a9e062eb19e8547e8b50434c7df52a8b96f9d0a254bc646d1f0d"
+
+  depends_on "pkg-config" => :build
+  depends_on "swig" => :build
+  depends_on "cairo"
+  depends_on "cfitsio"
+  depends_on "gsl"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "netpbm"
+  depends_on "numpy"
+  depends_on "python"
+  depends_on "wcslib"
+
+  resource "numpy" do
+    url "https://files.pythonhosted.org/packages/83/6b/d03277eacf113697675cd659086a4dcf9472108e2f1a83884c0271bdca46/numpy-1.15.3.zip"
+    sha256 "1c0c80e74759fa4942298044274f2c11b08c86230b25b8b819e55e644f5ff2b6"
+  end
+  resource "fitsio" do
+    url "https://files.pythonhosted.org/packages/9c/cb/f52534b71f4d99916723af2994898904015b9a1bf0286a165182d0374bbf/fitsio-0.9.11.tar.gz"
+    sha256 "a1196385ca7c42c93d9e53002d5ba574a8db452c3b53ef1189e2c150177d4266"
+  end
+
+  def install
+    ENV["NETPBM_INC"] = "-I#{Formula["netpbm"].opt_include}/netpbm"
+    ENV["NETPBM_LIB"] = "-L#{Formula["netpbm"].opt_lib} -lnetpbm"
+    ENV["SYSTEM_GSL"] = "yes"
+    ENV["PYTHON_SCRIPT"] = "#{libexec}/bin/python3"
+    ENV["PYTHON"] = "python3"
+
+    venv = virtualenv_create(libexec, "python3")
+    venv.pip_install resources
+
+    ENV["INSTALL_DIR"] = prefix
+    xy = Language::Python.major_minor_version "python3"
+    ENV["PY_BASE_INSTALL_DIR"] = "#{libexec}/lib/python#{xy}/site-packages/astrometry"
+    # FIXME: astrometry.net 0.76 does not get this variable from the environment.
+    # ENV["PY_BASE_LINK_DIR"] = "../libexec/lib/python#{xy}/site-packages/astrometry"
+    system "make"
+    system "make", "py"
+    system "make", "install"
+
+    # FIXME: work around PY_BASE_LINK_DIR
+    rm "#{bin}/plotann.py"
+    ln_sf "../libexec/lib/python#{xy}/site-packages/astrometry/blind/plotann.py", "#{bin}/plotann.py"
```

Comment:
Can you please try this instead of using `ln_sf`:

```suggestion
    bin.install_symlink libexec/"lib/python#{xy}/site-packages/astrometry/blind/plotann.py"
```

Also, the comment here should be changed to link to the upstream issue, indicating when it can be removed.

e.g.,
```
Work around for https://github.com/dstndstn/issues/###
```

Reply:
oops, should have replied here rather than below!  Once this FIXME is done, there will not be a need to special-case the symlink for plotann.py; the Makefile will just do the right thing with that PY_BASE_LINK_DIR variable set.